rental.rb

```diff
@@ -0,0 +1,11 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+
+  def initialize(book, person, date = DateTime.now())
+    @date = date
+    @book = book
+    @person = person
```

Comment:
Kindly make sure that the rental is adding itself to the @book to achieve the 'belongs-to' association.
Example : `book.rentals << self`

Also the same for the person